guides/common/modules/proc_installing-postgresql.adoc

```diff
@@ -7,6 +7,13 @@ You can install PostgreSQL using Red Hat Enterprise Linux Server 7 repositories
 
 include::snip_firewalld.adoc[]
 
+Note that the default configuration of external PostgreSQL needs to be adjusted to work with Satellite. The base recommended external database configuration adjustments are as follows:
```

Comment:
Always use macros for branding. See https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc and https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-satellite.adoc as well.
```suggestion
Note that the default configuration of external PostgreSQL needs to be adjusted to work with {Project}. The base recommended external database configuration adjustments are as follows:
```

Then there's also the placing. I think this belongs to the instructions. We already have one to modify `postgresql.conf` and change `listen_address`. Perhaps that's a better place to recommend it than the introduction text?

Reply:
In the ticket that was opened against this issue, it was specified to put this in section 4.13.1. PostgreSQL as an External Database Considerations, so I'm not sure if moving it would be in the scope of work I was given.

Reply:
@ekohl I don't see PostgreSQL anywhere else in this doc.  Where would you suggest putting the new information?

Reply:
There is the step `. Edit the `/var/opt/rh/rh-postgresql12/lib/pgsql/data/postgresql.conf` file:` if you scroll down a bit (line 27 before this patch, line 34 after).

Reply:
@ekohl Thank you!  I believe everything is corrected now.